command/update/command.go

```diff
@@ -269,8 +201,48 @@ func (c *Command) execute() error {
 			return microerror.MaskAny(err)
 		}
 
-		c.logger.Log("debug", fmt.Sprintf("updated endpoint for service '%s'", f.Kubernetes.Cluster.Service))
+		c.logger.Log("debug", fmt.Sprintf("added IP to endpoint of service '%s'", f.Kubernetes.Cluster.Service))
 	}
 
+	// Listen to OS signals issued by the Kubernetes scheduler.
+	listener := make(chan os.Signal, 2)
```

Comment:
Why is this buffered? Why `2`?

Reply:
Just copy/paste from microkit. You can send a termination signal and the program will initialize the shutdown procedure. When you send a signal again the program will exit immediately. This is also what Kubernetes is doing. First it sends SIGTERM and after some grace period (default 30 seconds AFAIK) it will send SIGKILL.

Reply:
> Package signal will not block sending to c: the caller must ensure that c has sufficient buffer space to keep up with the expected signal rate. For a channel used for notification of just one signal value, a buffer of size 1 is sufficient.

From `Notify` godoc. This makes sense.